src/agent/src/unix_main.cpp

```diff
@@ -23,6 +22,12 @@ int main(int argc, char* argv[])
         {
             StartAgent();
         }
+        else if (cmdParser.OptionExists("restart"))
+        {
+            StopAgent();
+            StartAgent();
+            LogInfo("Wazuh Agent restarted");
```

Comment:
If starting the wazuh agent fails the log would be:

"Wazuh Daemon start failed"

followed by:

"Wazuh Agent restarted"


Maybe this log is not needed... or we could just start with "Wazuh Agent restarting..." before the call to stop.

Reply:
Removed the comment altogether.